shell.nix

```diff
@@ -0,0 +1,12 @@
+# For compatibility with non-flake-enabled Nix versions
```

Comment:
...and certain tools like https://github.com/arrterian/nix-env-selector which don't support flakes yet